### ISSUE TITLE
B-Table select mode not working

### ISSUE THREAD

**Issue by memen45** [bootstrap-vue-next/bootstrap-vue-next#1376]
Trying to work with the b-table component and running into the following three issues concerning the select features.

1. SelectMode property seems to not work at all. The behaviour is always as 'single', only one row can be selected at a time. According to the typing: 

```ts
(property) selectMode?: "multi" | "range" | "single" | undefined
```

Using the `this.$refs.tabl.selectAllRows()` method, the emitted event is correct, and contain all items in the table. However, the rendering does not update. All rows still show as unselected.

2. The selectable column is inserted and shows a square for each row, both selected and unselected, as if the icon could not be found. 
![image](https://github.com/bootstrap-vue-next/bootstrap-vue-next/assets/27782135/6af8585e-5655-4c55-a300-21a62004297e)

3. I found the template slot `#select-cell`, and would like to place a checkmark there. How do you receive the row object in that template? 
For select-cell row is just an empty object `{}`:
```ts
<template #cell(name)="row"></template> <!-- normal column, row={ value, index, field ... etc } -->
<template #select-cell="row"></template> <!-- select column, row={} -->
```

**Comment by memen45** [comment]
Update:
1. Seems to be fixed now!
2. Is still an issue. There is no icon, only a change to blue square indicates that the row is selected.
3. Is still unresolved. How should the row element be accessed in the select-cell?

**Comment by VividLemon** [comment]
Fixed in 0.15. Closing for now
